src/main/java/seedu/address/storage/JsonAdaptedPublicAddress.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.addresses.Network;
+import seedu.address.model.addresses.PublicAddress;
+import seedu.address.model.addresses.PublicAddressFactory;
+
+/**
+ * Jackson-friendly version of {@link PublicAddress}
+ */
+class JsonAdaptedPublicAddress {
+
+
+    private final String address;
```

Comment:
Should standardise all variable names to publicAddress for all mentions of crypto address, since the actual field address already exists